specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -404,6 +404,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/locations/{location}/usagesByVMFamily": {
```

Comment:
Please comply with the ARM usages contract for this API. It is defined here - https://armwiki.azurewebsites.net/api_contracts/UsagesAPIContract.html

Reply:
@nkalpana Could you update per ARM usages api contract ?